config/tgl-cavs.toml

```diff
@@ -286,3 +286,23 @@ count = 10
 
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
+
+        [[module.entry]]
+        name = "AEC"
```

Comment:
AEC is a generic name, but there are lots of different versions and implementations. Can we try to have a clearer information on the provider of that solution?

Reply:
@plbossart  this is define in windows fw manifest and used by windows driver. I use it for compatibility. If we use other name we need to use different UUID

Reply:
Not talking about the UUID but the 'AEC' name. I am not sure why anyone thought it was a good idea to avoid mentioning this was an Intel IP.

@lgirdwood is this time to enforce a namespace or a convention to avoid issues like this?

Reply:
@plbossart I think this is in reference to the Google IP AEC